docs/lmos_protocol.md

```diff
@@ -0,0 +1,893 @@
+
+Here is the table and description for an **Agent** in the context of the **LMOS Protocol Vocabulary**:
+
+| **Vocabulary Term** | **Value** | **Assignment** | **Type** |
+|---------------------|-----------------|----------------|----------|
+| **@context**         | https://eclipse.dev/lmos/protocol/v1 | Mandatory | URI |
+| **@type**            | ns:Agent  | Mandatory | string
+
+
+Example:
+
+```
+{
+   "@context": [
+      "https://www.w3.org/2022/wot/td/v1.1",
+      {
+         "lmos": "https://eclipse.dev/lmos/protocol/v1",
+      }
+   ],
+   "@type": "lmos:Agent",
+   "id": "urn:uuid:6f1d3a7a-1f97-4e6b-b45f-f3c2e1c84c77",
+   "title": "Agent Name",
+   "lmos:metadata": {
+        "lmos:vendor": {
+            "lmos:name": "Deutsche Telekom AG",
+            "lmos:url": "https://telekom.de"
+        }
+   }
+}
+```
+
+Full example:
+
+```
+{
+   "@context": [
+      "https://www.w3.org/2022/wot/td/v1.1",
+      {
+         "lmos": "https://eclipse.dev/lmos/protocol/v1",
+      }
+   ],
+   "id": "urn:uuid:6f1d3a7a-1f97-4e6b-b45f-f3c2e1c84c77",
+   "title": "WeatherAgent",
+   "@type": "ex:Agent",
+   "links": [{
+      "rel": "service-doc",
+      "href": "https://weatherai.example.com/manual.pdf",
+      "type": "application/pdf",
+      "hreflang": "en"
+   }],
+   "lmos:metadata": {
+        "lmos:vendor": {
+            "lmos:name": "Deutsche Telekom AG",
+            "lmos:url": "https://telekom.de"
+        }
+   }
+    "securityDefinitions": {
+        "basic_sc": {
+            "scheme": "basic",
+            "in": "header"
+        }
+    },
+    "security": "basic_sc",
+    "properties": {
+        "modelConfiguration": {
+            "description": "Current configuration of the underlying LLM, including version, temperature, and maximum tokens.",
+            "type": "object",
+            "readOnly": true,
+            "properties": {
+                "modelName": {
+                    "type": "string",
+                    "description": "Name of the model in use, e.g., gpt-4o."
+                },
+                "temperature": {
+                    "type": "number",
+                    "description": "Temperature setting for controlling response randomness.",
+                    "minimum": 0,
+                    "maximum": 1
+                },
+                "maxTokens": {
+                    "type": "integer",
+                    "description": "Maximum number of tokens the model is allowed to generate."
+                }
+            },
+            "forms": [
+                {
+                    "op": "readproperty",
+                    "href": "https://weatherai.example.com/things/urn:uuid:6f1d3a7a-1f97-4e6b-b45f-f3c2e1c84c77/properties/modelConfiguration",
+                    "contentType": "application/json"
+                }
+            ]
+        }
+    },
+    "actions": {
+        "getWeather": {
```

Comment:
@RobWin why do TD.actions describe the agent tool specifictions?

Reply:
Actions describe the API "functions" of Agents or Tools. 
Next, we can harmonize actions within the LMOS framework by categorizing Agents into high-level types like Conversational Agents (ChatBots), Generative AI Agents (Videos, Images), and/or Analytical Agents.
Then each action will be described with a standardized schema/ontology for interoperability.